### PR TITLE
160 refactor graphs for json/ input graphs

### PR DESCRIPTION
**Pull request by kdumais111**
Still much to do but the basic graphs are there and in the correct color scheme

Current issues:
- There is no portland bus data
- The weekend week day data is duplicative for top 10
- Somehow when i merged with main the color key from the legend for the heat map stopped working. 
- x axis bleed

Screenshot attached

### CODE REVIEW

**Comment by kdumais111** [comment]
Screen Shot 2024-05-22 at 2 12 11 AM
Screen Shot 2024-05-22 at 4 52 49 AM
Screen Shot 2024-05-22 at 5 24 27 AM

File: app/route_rangers_api/static/styles.css [hunk 1]
```diff
@@ -670,6 +670,16 @@ h2 {
   font-size-adjust: 2rem;
   color: #555;
 }
+h3 {
+  /* margin: 100px; */
```
Comment (lisette-solis) [inline app/route_rangers_api/static/styles.css:674]: delete these notes

**Review by lisette-solis — CHANGES_REQUESTED** [review]
Looks great! thank you for getting all these done. The only two things I would suggest is adding titles to the graphs (I can help on this) and making the graphs a little smaller.

File: app/route_rangers_api/views.py [hunk 2]
```diff
@@ -55,7 +63,7 @@ def about(request):
 
 
 # caching for 6 hours since the data doesn't change often
-@cache_page(60 * 60 * 6)
+# @cache_page(60 * 60 * 6)
```
Comment (meganhmoore) [inline app/route_rangers_api/views.py:66]: uncomment

**Review by lisette-solis — APPROVED** [review]

**Pull request merged**